includes/ConsoleTasks/ClearOAuthDataTask.php

```diff
@@ -19,7 +20,7 @@ public function execute()
         $database = $this->getDatabase();
 
         $users = UserSearchHelper::get($database)->inIds(
-            $database->query('SELECT user FROM oauthtoken WHERE type = \'access\'')->fetchColumn()
+            $database->query('SELECT user FROM oauthtoken WHERE type = \'access\'')->fetchAll(PDO::FETCH_COLUMN)
```

Comment:
Nit: [PHP documentation](https://www.php.net/manual/en/pdostatement.fetchall.php) suggests that the `$column` parameter is required when using `PDO::FETCH_COLUMN`. But it seems to work regardless!